docs/netlify.toml

```diff
@@ -1197,6 +1197,12 @@
   to = "/quickstart/428201/custom-function"
   status = 301
 
+[[redirects]]
+  # redirect schema to the versioned one
+  from = "/reference/dagger.schema.json"
+  to = "/reference/schemas/v0.14.0/dagger.schema.json"
```

Comment:
shall we remove this redirection altogether and force that all `/referece/*` URLs have to be versioned? I'd prefer to bite the bullet now instead of having to find a way to automatically update this redirect each time a version is released. 

WDYT?

Reply:
This redirection is just so that anyone who was using the schema address before now actually gets redirected to a versioned one (so we don't have to keep publishing that file).

We don't need to automatically update it - we just keep it pinned to v0.14.0 - the main schema address to use will always have versions in it, it's just to avoid breaking users who currently are using it.

Reply:
> This redirection is just so that anyone who was using the schema address before now actually gets redirected to a versioned one (so we don't have to keep publishing that file).

Yep, that's what I imagined, thing is that we'll have to remember that this particular URL  points to v0.14.0's schema which is super confusing to me.  I'd expect the URL **without** the version to either redirect you to the `latest` schema, or return 404. That's mostly the reason in my comment above I've suggested removing the redirection altogether and to break it for some users in lieu of better consistency.

Reply:
I agree with @marcosnils. I would either remove this redirect entirely, or adjust it to point to a moving `latest`.